stories/Molecules/Text/ProjectDetails/ProjectDetails.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './projectdetails.scss';
+
+export const Projectdetails = ({ startName, startText, endName, endText, statusName, statusText }) => {
+    return (
+        <div className='projectdeta'>
+            <div className='projectdeta__col'>
+                <span>{startName}</span>
+                <p>{startText}</p>
```

Comment:
@Shubham-Sharma-77 it can't be p tag and we can also give it span and add the classes on span and on the basis of classes we can add the styling